scripts/zowe-install-zlux.sh

```diff
@@ -54,11 +56,11 @@ cp -f ${INSTALL_DIR}/files/zlux/config/zluxserver.json zlux-app-server/defaults/
 cp -f ${INSTALL_DIR}/files/zlux/config/plugins/* zlux-app-server/defaults/plugins
 
 echo "Unpax zssServer " >> $LOG_FILE
-cd zlux-app-server
+cd ${ZSS_COMPONENT_DIR}
 pax -r -px -f $INSTALL_DIR/files/zss.pax bin
-cd bin
-extattr +p zssServer
-cd ../..
+extattr +p bin/zssServer
+cp -r ${ZSS_COMPONENT_DIR}/bin/z* ${APP_SERVER_COMPONENT_DIR}/share/zlux-app-server/bin
```

Comment:
oh one more thing... extattr +p is not portable. Copies lose it, so you need `extattr +p ${APP_SERVER_COMPONENT_DIR}/share/zlux-app-server/bin/zssServer`